src/UsgsAstroPlugin.cpp

```diff
@@ -273,97 +293,111 @@ std::string UsgsAstroPlugin::convertISDToModelState(const csm::Isd &imageSupport
 csm::Model *UsgsAstroPlugin::constructModelFromISD(const csm::Isd &imageSupportDataOriginal,
                                               const std::string &modelName,
                                               csm::WarningList *warnings) const {
-    std::string stringIsd = loadImageSupportData(imageSupportDataOriginal);
-
-    if (modelName == UsgsAstroFrameSensorModel::_SENSOR_MODEL_NAME) {
-      UsgsAstroFrameSensorModel *model =  new UsgsAstroFrameSensorModel();
-      try {
-        model->replaceModelState(model->constructStateFromIsd(stringIsd, warnings));
-        std::shared_ptr<spdlog::logger> logger = model->getLogger();
-        if (logger) {
-          logger->info("Constructed model: {}", modelName);
-        }
-      }
-      catch (std::exception& e) {
-        csm::Error::ErrorType aErrorType = csm::Error::SENSOR_MODEL_NOT_CONSTRUCTIBLE;
-        std::string aMessage = "Could not construct model [";
-        aMessage += modelName;
-        aMessage += "] with error [";
-        aMessage += e.what();
-        aMessage += "]";
-        std::string aFunction = "UsgsAstroPlugin::constructModelFromISD()";
-        throw csm::Error(aErrorType, aMessage, aFunction);
+  MESSAGE_LOG("Running constructModelFromISD");
+  std::string stringIsd = loadImageSupportData(imageSupportDataOriginal);
+  
+  MESSAGE_LOG("ISD String: {}", stringIsd);
+  if (modelName == UsgsAstroFrameSensorModel::_SENSOR_MODEL_NAME) {
+    UsgsAstroFrameSensorModel *model =  new UsgsAstroFrameSensorModel();
+    try {
+      MESSAGE_LOG("Trying to construct a UsgsAstroFrameSensorModel");
+      model->replaceModelState(model->constructStateFromIsd(stringIsd, warnings));
+      std::shared_ptr<spdlog::logger> logger = model->getLogger();
+      if (logger) {
+        logger->info("Constructed model: {}", modelName);
```

Comment:
Why does this not use the logging macro?

Reply:
Ah! This was the only part of the plugin that was previously logged before I added to the Macro. Good catch, thanks!